### PR TITLE
Handle requests not using RBAC

### PR DESCRIPTION
**Pull request by JumasJM**
Type: Improvement
Signed-off-by: jmasar <jmasar@frinx.io>

### CODE REVIEW

**Review by Vasyl-9 — APPROVED** [review]

File: rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java [hunk 1]
```diff
@@ -73,9 +73,11 @@ public void doFilter(
 
         HttpServletRequest request = (HttpServletRequest) servletRequest;
         HttpServletResponse response = (HttpServletResponse) servletResponse;
+        List<String> noRbacValues = List.of("/health", "/v3/api-docs");
 
-        final String healthCheck = "health";
-        if (request.getRequestURI().contains(healthCheck)) {
+        boolean isRbacless =
+                noRbacValues.stream().anyMatch(value -> value.equals(request.getRequestURI()));
```
Comment (Vasyl-9) [inline rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java:79]: Change to startWith() for capture sub-requests or query parameters.

File: rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java [hunk 2]
```diff
@@ -73,9 +73,11 @@ public void doFilter(
 
         HttpServletRequest request = (HttpServletRequest) servletRequest;
         HttpServletResponse response = (HttpServletResponse) servletResponse;
+        List<String> noRbacValues = List.of("/health", "/v3/api-docs");
 
-        final String healthCheck = "health";
-        if (request.getRequestURI().contains(healthCheck)) {
+        boolean isRbacless =
```
Comment (Vasyl-9) [inline rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java:78]: rename variable to ignoreRbac or something similar.

**Review by Vasyl-9 — CHANGES_REQUESTED** [review]

File: rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java (hunk 1 above)
Comment (JumasJM) [reply rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java]: Resolved.

File: rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java (hunk 2 above)
Comment (JumasJM) [reply rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java]: Renamed.

**Review by Vasyl-9 — APPROVED** [review]

**Pull request merged**